.github/workflows/new_issue_to_pyrus_task.yaml

```diff
@@ -0,0 +1,85 @@
+name: "Create Pyrus task from issue"
+
+on:
+  workflow_call:
+    inputs:
+      PACKAGE_NAME:
+        type: string
+        required: true
+      ISSUE_TITLE:
+        type: string
+        required: true
+      ISSUE_BODY:
+        type: string
+        required: true
+      ISSUE_LINK:
+        type: string
+        required: true
+    secrets:
+      LOGIN:
+        required: true
+      SECURITY_KEY:
+        required: true
+
+jobs:
+  create-pyrus-task:
+    runs-on: ubuntu-latest
+    env:
+      LOGIN: ${{ secrets.LOGIN }}
+      SECURITY_KEY: ${{ secrets.SECURITY_KEY }}
+    steps:
+      - id: auth
+        run: |
+          # Authorization with receiving a token
+          response=$(curl -X POST \
+          https://api.pyrus.com/v4/auth \
+          -H 'Content-Type: application/json' \
+          -d '{
+            "login": "'"$LOGIN"'",
+            "security_key": "'"$SECURITY_KEY"'"
+            }'
+          )
+
+          # Check for the absence of 'access_token' field in the response
+          if ! echo "$response" | jq -e '.access_token' > /dev/null; then
+            echo "Error in API request. Response: $response"
```

Comment:
I think it’s worth making sure that this step fails in case of an error at this stage

Reply:
same